wpeverest/everest-forms
# Feature - Divi Builder Compatibility

### All Submissions:






### Changes proposed in this Pull Request:



Previously, There was no module in Divi builder for Everest Forms for its compatibility. This PR adds this feature to use the Everest Forms directly from the Divi module.

Closes # .

### How to test the changes in this Pull Request:

1. Install the **Divi theme**. (It is mandatory)
2. Go to **Everest Forms Dashbaord > Features > Enable Divi Module**
3. Create the **Form** or you can also use the **existing form**.
4. Now go to **Pages >Add New**
5. Click on Use **Divi Builde**r button
6. Now search for **Everest Forms Module**
7. Select the desired form and perform the customization you want to have
8. Now on the bottom of page you can see three dots you can see **Save** button
9. Click on the **Save** button
10. Now check whether whether the form is rendering and working accordingly or not. 

### Types of changes:




### Other information:




### Changelog entry

>Feature - Divi Builder Compatibility.
